model/src/main/proto/exploration.proto

```diff
@@ -0,0 +1,145 @@
+syntax = "proto3";
+
+import "subtitled_html.proto";
+
+package model;
+
+option java_package = "org.oppia.app.model";
+option java_multiple_files = true;
+
+// Data for a single exploration.
+// Maps from: data/src/main/java/org/oppia/data/backends/gae/model/GaeExploration.kt
+message Exploration {
+  repeated State states = 1;
```

Comment:
Shouldn't this be a map, or should State maybe have a name? I don't think there's currently a way to reference states with how this structure is set up since state_id is missing.

Reply:
Done